src/site/SW.yotsuba.coffee

```diff
@@ -27,6 +27,12 @@ SW.yotsuba =
       uniqueIDRoot: '.posteruid'
       uniqueID:  '.posteruid > .hand'
       capcode:   '.capcode.hand'
+      capcodeJumperRoot: '.capcodeJumper'
+      capcodeJumperPrev: '.capcodeJumper > .prev'
+      capcodeJumperNext: '.capcodeJumper > .next'
+      uniqueIDJumperRoot: '.uniqueIDJumper'
+      uniqueIDJumperPrev: '.uniqueIDJumper > .prev'
+      uniqueIDJumperNext: '.uniqueIDJumper > .next'
```

Comment:
These don't belong here; this is only for parsing the original HTML of posts. You can add these properties to the post's `nodes` object if you want, but you need to do it from `PostJumper.node` or one of the functions it calls. The `isClone` property may be useful to you. It's wasteful to check for these at initial parse time when we know they're not there, plus all of this would need to be duplicated to `SW.tinyboard` and any others that get added.